scripts/process_data.py

```diff
@@ -264,6 +278,12 @@ def main(self) -> None:
             self.data, image_dir=image_dir, num_frames_target=self.num_frames_target, verbose=self.verbose
         )
 
+        # Generate planar projections if equirectangular
+        if self.camera_type == "equirectangular":
+            image_dir = process_data_utils.generate_planar_projections_from_equirectangular(
```

Comment:
There's an issue with where the images are being put at the end of the video processing